src/coverlet.core/DependencyInjection.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Coverlet.Core
 {
-    internal static class DependencyInjection
+    public static class DependencyInjection
```

Comment:
The only thing I need to do is to expose container to be used by console/task/collector without expose internal implementation.